### ISSUE TITLE
AdditionalAddressLines

### ISSUE THREAD

**Issue by mrphus** [w3stling/lei#1]
Thanks for this library - saved me some time in coding. 

I just have an issue with the addresses - according to the standard the addresses can also have an "AdditionalAddressLine". 

Example would be: https://leilookup.gleif.org/api/v2/leirecords?lei=549300BORA0TFB7DYG22

I just debugged the class 

`com.apptastic.lei.LeiLookup.parseAddress(JsonReader)`

and it seems that in line 311 he is simply ignoring the key-name "AdditionalAddressLine". 

Fix would be to add a String additionalAddressLine in class `com.apptastic.lei.Lei.Address` and some added mappings in `LeiLookup.parseAddress`:

``` 
else if ("AdditionalAddressLine".equals(name)) {
     address.additionalAddressLine = getValue(jsonReader);
}
```

**Comment by w3stling** [comment]
Added method `getAdditionalAddressLine()` to `Lei.Address` class. Available in release [v1.2.1](https://bintray.com/beta/#/apptastic/maven-repo/lei/1.2.1?tab=overview)

Example:
`List<String> additionalAddressLine = legalAddress.getAdditionalAddressLine();`

**Comment by mrphus** [comment]
Works fine, thanks for the fast response!
